terraform/core/47-mwaa.tf

```diff
@@ -213,7 +213,7 @@ resource "aws_mwaa_environment" "mwaa" {
   tags                  = module.tags.values
 
   airflow_configuration_options = {
-    "core.default_timezone"               = "Europe/London"
+    "core.default_timezone"               = "utc"
```

Comment:
will this change the displayed run time and affect scheduling in the UI?

could you not handle the utc time in the task?

Reply:
This does not impact the UI. It supports both local time and UTC, with two options available. Users can always click the button to adjust based on personal preference.
![image](https://github.com/user-attachments/assets/18e24161-74a6-474e-b6e3-19c59b639a9d)

Yes, it will impact the existing DAGs since they use the Airflow system timezone.

Yes, I can. I am fine with not changing from London to UTC. Let me remove this part and only change the maintenance window.